package.json

```diff
@@ -97,10 +97,10 @@
     "extends": "@istanbuljs/nyc-config-typescript",
     "all": true,
     "check-coverage": true,
-    "branches": 90,
-    "lines": 85,
-    "functions": 90,
-    "statements": 85,
+    "branches": 50,
```

Comment:
note: we are lowering the test coverage temporarily for demo purposes, another ticket will be bringing the coverage back up!